frontend/src/main/scala/bloop/data/SourcesGlobs.scala

```diff
@@ -0,0 +1,69 @@
+package bloop.data
+
+import java.nio.file.FileSystems
+import java.nio.file.PathMatcher
+import java.nio.file.Path
+import bloop.config.Config
+import scala.util.Properties
+
+case class SourcesGlobs(
+    directory: Path,
```

Comment:
Should this be `AbsolutePath` instead? I tried doing that refactoring but ended up only only unwrapping it at every call-site

Reply:
Done, I refactored to use `AbsolutePath`.